### PR TITLE
Polish chat inbox locked out row

### PR DESCRIPTION
**Pull request by nathunsmitty**
Adjusts the line height to make the chat locked out row a little nicer.

screen shot 2018-07-23 at 3 56 06 pm

Also some general style refactoring + adding some inbox rows to storybook.

### CODE REVIEW

File: shared/styles/css.js.flow [hunk 1]
```diff
@@ -265,7 +265,7 @@ export type _StylesDesktop = {|
   left?: Dimension,
   letterSpacing?: string,
   lineBreak?: string,
-  lineHeight?: string,
+  lineHeight?: Dimension,
```
Comment (nathunsmitty) [inline shared/styles/css.js.flow]: Guessing we don't want to type line heights as strings?

Comment (chrisnojima) [reply shared/styles/css.js.flow]: i mentioned this in the other pr i think we don't

File: shared/react-native/ios/Keybase/Storybook.m [hunk 2]
```diff
@@ -24,7 +24,7 @@ + (BOOL)requiresMainQueueSetup
 - (NSDictionary *)constantsToExport
 {
   // Set this to true to enable storybook mode
-  return @{@"isStorybook": @false};
+  return @{@"isStorybook": @true};
```
Comment (chrisnojima) [inline shared/react-native/ios/Keybase/Storybook.m]: 🔥

**Comment by chrisnojima** [comment]
lets just rename the bottom part to say 'You are currently locked out' or something

**Comment by cecileboucheron** [comment]
Even shorter: `You are locked out.`

**Comment by nathunsmitty** [comment]
Also caught a bug where the "Rekey Needed" alert was centering itself in mobile and taking up the full width on desktop. It should behave just like the "Reset" alert now.
screen shot 2018-07-23 at 4 56 10 pm

**Comment by nathunsmitty** [comment]
And here's how the one-line locked out text looks! I think this is ready for another look
screen shot 2018-07-23 at 4 56 02 pm

**Comment by nathunsmitty** [comment]
@keybase/react-hackers 👀?

File: shared/chat/inbox/row/small-team/bottom-line.js [hunk 3]
```diff
@@ -159,53 +114,74 @@ class BottomLine extends PureComponent<Props> {
...
+    paddingLeft: globalMargins.xtiny,
+    paddingRight: globalMargins.xtiny,
+  },
+  rekeyNeededText: platformStyles({
+    common: {
+      color: globalColors.white,
+    },
+    isElectron: {
+      fontSize: 11,
+      lineHeight: '13px',
```
Comment (chrisnojima) [inline shared/chat/inbox/row/small-team/bottom-line.js]: this should just be  13, platformstyles does the cast for you

File: shared/chat/inbox/row/small-team/bottom-line.js [hunk 4]
```diff
@@ -159,53 +114,74 @@ class BottomLine extends PureComponent<Props> {
...
+    },
+    isMobile: {
+      fontSize: 12,
+      lineHeight: 14,
+    },
+  }),
+  youAreResetText: platformStyles({
+    isElectron: {
+      fontSize: 12,
+      lineHeight: '13px',
```
Comment (chrisnojima) [inline shared/chat/inbox/row/small-team/bottom-line.js]: same here

File: shared/chat/inbox/row/small-team/bottom-line.js [hunk 5]
```diff
@@ -230,8 +206,18 @@ const styles = styleSheetCreate({
...
-  bottomLineBox: {
+  contentBox: {
     width: '100%',
   },
+  alertMeta: platformStyles({
+    common: {
+      alignSelf: 'center',
+      marginRight: 6,
+    },
+    isElectron: {},
```
Comment (chrisnojima) [inline shared/chat/inbox/row/small-team/bottom-line.js]: 🔥

File: shared/chat/inbox/row/small-team/bottom-line.js (hunk 3 above)
Comment (nathunsmitty) [reply shared/chat/inbox/row/small-team/bottom-line.js]: that gives a flow error because lineHeight has to be a string

Comment (chrisnojima) [reply shared/chat/inbox/row/small-team/bottom-line.js]: that type is wrong then, it should be Dimension

**Pull request merged**